Source/Core/Core/HW/DVDInterface.cpp

```diff
@@ -236,11 +255,16 @@ static u32  g_ErrorCode = 0;
 static bool g_bDiscInside = false;
 bool g_bStream = false;
 static bool g_bStopAtTrackEnd = false;
-static int  tc = 0;
+static int  finish_execute_command = 0;
+static int  finish_execute_read_command = 0;
 static int  dtk = 0;
 
 static u64 g_last_read_offset;
 static u64 g_last_read_time;
+static u64 g_last_command_time;
+
+static std::deque<DVDReadCommand> read_commands_to_execute;
+static u64 next_read_command_id;
```

Comment:
It's theoretically possible to send a DI command before the last one is done, but that doesn't make any sense, so it's not really something we have to care about. So, hmm... There could be a queue in Device_DI instead, where incoming IPC commands wait until DVDInterface isn't executing anything. That would also make it possible to get rid of g_last_command_time from DVDInterface, so it sounds like a good idea in overall.